File: Fetch stations/FetchStations/Networking/Place.m
//
//  Place.m
//  FetchStations
//
//  Created by Waylan Sands on 3/5/2023.
//

#import <Foundation/Foundation.h>
#import "Place.h"

@implementation Place

- (instancetype)initWithDictionary:(NSDictionary *)dictionary {
    self = [super init];
    if (self) {
        self.name = dictionary[@"name"];
        self.type = dictionary[@"type"];
        self.id = dictionary[@"id"];
    }
    return self;
}

@end

File: FetchStations/Models/Place.swift
//
//  Place.swift
//  FetchStations
//
//  Created by Waylan Sands on 3/5/2023.
//

import Foundation

struct Place: Codable {
    let location: Location
    let name: String
    let type: String
    let id: String
}

struct Location: Codable {
    let lat: Double
    let lng: Double
}
